docs/node/development/contributing.md

```diff
@@ -1,168 +1,55 @@
-
-Good
-```cpp
-struct A {
-    char *message;
-    int length;
-}
-```
-
-Bad:
-```cpp
-struct A {
-    char *msg;
-    int len;
-}
-```
-
-### Placement of spaces
-
-In variable declarations (including in function parameters) we place the asterisk next to the variable name.
-
-Good:
-```cpp
-int *number;
-```
-
-Bad:
-```cpp
-int * number;
-int* number;
-```
-
-In function return values we place the asterisk between spaces
-
-Good:
-```cpp
-void * my_thread()
-{
-    
-}
-```
-
-Bad:
-```cpp
-void *mythread()
-{
-    
-}
-
-void * mythread()
-{
-    
-}
-```
-
-In expressions we place spaces between most elements but not between parantheses:
-
-Good:
-```cpp
-int a = ((5 * 10) + 10 ) % 22;
-```
-
-Bad:
-```cpp
-int a = ((5*10)+10)%22;
-```
-
-We also put spaces in front / after most keywords such as `if`, `while`, ...
-
-Good:
-```cpp
-for (int a = 0; a < 1; a++) {
-    ...
-}
-```
-
-Bad:
-```cpp
-for(int a = 0; a < 1; a++) {
-    ...
-}
-```
-
-### Line length
-
-Try to limit line length to about 160 characters.
-
-### Indention
-
-We use tabs for indention which should be displayed as 8 spaces by your editor.
-
-### Brace placement
-
-For everything except functions we place the opening brace in the same line:
-
-Good:
-```cpp
-struct A {
-    enum {
-        A,
-        B
-    } choice;
-}
-
-if (true) {
-    
-}
-```
-
-Bad:
-```cpp
-struct A
-{
-    enum
-    {
-        A,
-        B
-    } choice;
-}
-
-if (true)
-{
-    
-}
-```
-
-However, in functions we place the brace in a new line:
-
-```cpp
-void main()
-{
-    
-}
-```
-
-### #define and Macro Names
-
-Are always written in `UPPERCASE`.
-
-Good:
-```cpp
-#define HUGEPAGESIZE (1 << 22)
-#define MAX(a, b) (a > b ? a ? b)
-```
-
-Bad:
-```cpp
-#define max(a, b) (a > b ? a ? b)
-```
-
-## Always work on feature branches
-
-Please branch from `develop` to create a new _feature_ branch.
-
-Please create a new _feature_ branch for every new feature or fix.
-
-## Make the CI happy :-)
-
-Only branches which pass the CI can be merged.
+All contributions to VILLASframework are welcome! We are happy to receive your pull requests.
+
+To keep our codebase maintainable and readable, we developed a few guidelines that we ask you to follow when contributing code.
+The point of this document is to have clear rules that if followed should guarantee that a PR is accepted without much discussion.
+Reviewers should only enforce these rules.
+
+Discussions on the contributor guidelines should happend the in the [documentation repository](https://github.com/VILLASframework/documentation) and not in unrelated PRs.
+
+As the codebase and these guidelines evolve, the codebase may not always be in full compliance with the guidelines.
+Existing code should not be taken as and example for new code.
+PRs that update the style of existing code should be made separate from PRs that change or add code.
+
+
+## VILLASframework Coding Guidelines
+1. Automated formatting tools
```

Comment:
Can we include a reference to the [LLVM coding standards](https://llvm.org/docs/CodingStandards.html) as a base?

Reply:
I had that in there before, but removed it to avoid confusing / discussions. They are very strict in some areas and we do some things differently to them. Maybe I add it at the bottom framed more as a reference rather than what we base our guidelines on.